components/SubmissionFlow.tsx

```diff
@@ -35,6 +39,18 @@ export const SubmissionFlow: React.FC = (props) => {
       if (!authenticated) {
         throw "Not Authenticated";
       }
+      
+      if (!!fileSelected){
+        submission.mediaURI = await uploadFiles({
```

Comment:
<img width="483" alt="image" src="https://user-images.githubusercontent.com/19307482/179173149-13a4e186-e36b-44c1-8673-59f29e9db02f.png">
I am getting this error because the mediaURI is missing from the request. But I am not sure how to include it.

Reply:
Don't